<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CredencialesApiWoo</class>
 <widget class="QDialog" name="CredencialesApiWoo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credenciales API WooCommerce</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../iconos.qrc">
    <normaloff>:/assets/icons/actualizar.png</normaloff>:/assets/icons/actualizar.png</iconset>
  </property>
  <property name="styleSheet">
   <string>
QDialog {
    background-color: #ffffff;
    font-family: &quot;Segoe UI&quot;;
    font-size: 10pt;
}

QLabel {
    color: #222222;
}

QLineEdit {
    border: 1px solid #cfd8dc;
    border-radius: 6px;
    padding: 6px 8px;
    background-color: #ffffff;
}

QLineEdit:focus {
    border: 1px solid #2979ff;
}

QCheckBox {
    spacing: 6px;
}

/* ---------- BOTONES (BASE) ---------- */
QPushButton {
    border-radius: 8px;
    padding: 6px 16px;
    font-weight: bold;
    color: white;
}

/* BOTÓN PRIMARIO - GUARDAR */
QPushButton#btnGuardar {
    background-color: #1e73f1;
}
QPushButton#btnGuardar:hover {
    background-color: #1558c0;
}
QPushButton#btnGuardar:pressed {
    background-color: #0d47a1;
}

/* BOTÓN SECUNDARIO - PROBAR */
QPushButton#btnProbarConexion {
    background-color: #90caf9;
    color: #0d47a1;
}
QPushButton#btnProbarConexion:hover {
    background-color: #64b5f6;
}
QPushButton#btnProbarConexion:pressed {
    background-color: #42a5f5;
}

/* BOTÓN CERRAR */
QPushButton#btnCerrar {
    background-color: #9e9e9e;
}
QPushButton#btnCerrar:hover {
    background-color: #757575;
}
QPushButton#btnCerrar:pressed {
    background-color: #616161;
}

/* ---------- SEPARADOR ---------- */
QFrame#line {
    color: #e0e0e0;
}
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">
   <item>
    <widget class="QLabel" name="labelTitulo">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string>
background-color: #1e73f1;
color: white;
padding: 10px;
border-radius: 8px;
      </string>
     </property>
     <property name="text">
      <string>WooCommerce – Credenciales API</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkCredencialesCargadas">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Credenciales cargadas.</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="line">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="verticalSpacing">
      <number>14</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelStoreUrl">
       <property name="text">
        <string>E-COMMERCE URL</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditStoreUrl"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelConsumerKey">
       <property name="text">
        <string>Consumer Key</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditConsumerKey">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelConsumerSecret">
       <property name="text">
        <string>Consumer Secret</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditConsumerSecret">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkVerCredenciales">
     <property name="text">
      <string>Ver credenciales</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnProbarConexion">
       <property name="text">
        <string>Probar conexión</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCerrar">
       <property name="text">
        <string>Cerrar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGuardar">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../iconos.qrc"/>
 </resources>
 <connections/>
</ui>
